### ISSUE TITLE
Add documentation about correct way of handling non-existent content in importer function

### ISSUE THREAD

**Issue by ubenzer** [sass/node-sass#697]
Hello,

I am trying to use node-saas with custom importer function. The reason I am doing this is, I don't directly read from file, instead I am compiling saas from in-memory strings. 

In importer function,
in case I can't find the partial how should I handle this error so it fails gracefully?  
https://github.com/sass/node-sass#importer-starting-from-v2

Should I `return null;` or `done(null);` ? Or should I simply throw a `new Error("File not found")` ?

**Comment by am11** [comment]
Would you like to send a PR based on your findings and the discussion that transpired here: https://github.com/sass/node-sass/pull/615?

**Comment by am11** [comment]
This is fixed by c481f30.

The new implementation (https://github.com/sass/node-sass/pull/817/) lets you use the API as follow:

```javascript
var sass = require('node-sass');

var options = {
  file: 'path/to/file.scss',
  importer: functions (url, prev, done) {
    
      try {
        var response = {
        file: processPaths(url, prev)
        };
        done(response)
      } catch (_) {
        done(new Error('my faviourite error message'));
      }
  }
};

sass.render(options, function (err, response) {
  if (err) {
    console.error(err); return;
  }
})
```
